templates/common/infra/bicep/core/database/mysql/mysql.bicep

```diff
@@ -0,0 +1,101 @@
+@description('Server Name for Azure database for MySQL')
+param name string
+@description('Location for all resources.')
+param location string = resourceGroup().location
+param tags object = {}
+
+param keyVaultName string
+@description('Database administrator login name')
+@minLength(1)
+param mysqlAdminName string
+
+param mysqlAdminPassKey string = 'MYSQL-PASS'
+
+@description('Database administrator password')
+@minLength(8)
+@secure()
+param mysqlAdminPassword string
+
+@description('Azure database for MySQL sku name ')
+param skuName string = 'Standard_B1s'
+
+@description('Azure database for MySQL storage Size ')
+param StorageSizeGB int = 20
```

Comment:
casing

Reply:
These case-problematic variables are decompiled by `az bicep decompile` from the ARM template json of an existing MySQL server. I will fix it anyway.